sql/parser2/scan_test.go

```diff
@@ -40,11 +40,9 @@ func TestScanner(t *testing.T) {
 		{`>`, []int{'>'}},
 		{`>=`, []int{GREATER_EQUALS}},
 		{`=`, []int{'='}},
-		{`=>`, []int{EQUALS_GREATER}},
```

Comment:
Nope. You're probably thinking of `>=`. Search the postgres grammar I linked to. There are only 2 references to `EQUALS_GREATER`.